dhall/src/Dhall.hs

```diff
@@ -1582,33 +1582,69 @@ instance Inject Int where
 
         declared = Integer
 
+{-| Injecting `Word` types gives `Natural`
+>>> embed inject (12 :: Word)
+NaturalLit 12
+-}
+
+instance Inject Word where
+    injectWith _ = InputType {..}
+      where
+        embed = NaturalLit . fromIntegral
+
+        declared = Natural
+
+{-| Injecting `Word8` types gives `Natural`
+>>> import Data.Word (Word8)
```

Comment:
Can you move these imports to a setup section like this one:

https://github.com/dhall-lang/dhall-haskell/blob/bf40ec52a1d33bf2688d83fdee6bb334dd706cf8/dhall/src/Dhall/Map.hs#L677-L683

Reply:
Where? Just before the instances? At the very top?

Reply:
So far we've put these at the end of the files, which seems fine to me.